youthversity/backend/templates/legal/report_comment.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html' %}
+{% block content %}
+<div class="header">
+  <h1>Reports</h1>
+</div>
+
+<div class="content">
+
+  <form action="/comment/{{ id }}/report/" method="post" class="pure-form pure-form-aligned">

```

Comment:
hardcoded url are not advised. Instead, the use of django's url engine is recommended. Among others one advantage is, that when using the engine, the url paths can be easily changed without any impact. So instead of `"/comment/{{ id }}/report/"` it would be better to use "url 'comment_report' id". (assuming that `'comment_report'` is the name of that endpoint as defined in `urls.py`.

Reply:
thanks for the hint, should be fixed now at (nearly)* all positions 
*I'm not quite sure if I've found all